frontend/src/pages/ConnectedPages/Leaderboard/index.tsx

```diff
@@ -23,46 +18,22 @@ const Leaderboard: React.FC = () => {
   >(null);
 
   useEffect(() => {
-    // TODO: Request Leaderboard from database
-    // GET "/leaderboard"
-    // const requestLeaderboard: IRequestLeaderboard = await axios.get("http://localhost:8888/leaderboard")
-    // data: ILeaderboard[];
-
-    const requestLeaderboard: IRequestLeaderboard = {
-      data: [
-        {
-          id: "difh",
-          name: "isdhsij",
-          nbGames: 27,
-          score: 9584,
-          successRate: 0.7
-        },
-        {
-          id: "weovh",
-          name: "xc ijsd",
-          nbGames: 384,
-          score: 38452,
-          successRate: 0.3
-        },
-        {
-          id: "csoj",
-          name: "svsc oc",
-          nbGames: 4975,
-          score: 12938,
-          successRate: 0.5
-        }
-      ],
-      complete: true,
-      error: false
+    requestLeaderboard();
+  }, []);
+
+  const requestLeaderboard = async () => {
+    const request = {
+      method: "get" as Method,
+      url: "/scores/leaderboard",
+      data: {}
     };
 
-    if (
-      requestLeaderboard.complete === true &&
-      requestLeaderboard.error === false
-    ) {
-      setLeaderboardTable(requestLeaderboard.data);
+    const response = await axiosRequest(request);
```

Comment:
You can type the response to know what you have if you want but it is not necessary.